omnibus/files/private-chef-ctl-commands/open_source_chef12_upgrade.rb

```diff
@@ -71,7 +71,7 @@ def download_chef11_data_setup
 
   def download_chef11_data_download(chef11_data_dir, key_file)
 
-    wait_for_ready_server("Chef 11")
+    wait_for_ready_server("Chef 11") unless @options.skip_setup
```

Comment:
This change appears unrelated to adding the sql options. Was this an oversight when the skip setup option was added?